addons/mail/models/res_config_settings.py

```diff
@@ -43,3 +46,40 @@ def _compute_fail_counter(self):
             ('date', '>=', previous_date),
             ('state', '=', 'exception'),
         ])
+
+    def open_general_mail_layout(self):
+        layout = self.env.ref('mail.mail_notification_paynow')
```

Comment:
Otherwise it will show a traceback instead of your UserError

```suggestion
        layout = self.env.ref('mail.mail_notification_paynow', raise_if_not_found=False)
```